### PR TITLE
Remove superfluous comment

### PR DESCRIPTION
**Pull request by ryanlntn**
@jamonholmgren @kevinvangelder

### CODE REVIEW

File: lib/ProMotion/iap.rb [hunk 1]
```diff
@@ -124,50 +164,6 @@ def transaction_product_id(transaction)
       transaction.respond_to?(:payment) ? transaction.payment.productIdentifier : "all"
     end
 
-    public
```
Comment (jamonholmgren) [inline lib/ProMotion/iap.rb:127]: I could be swayed on this, but I liked grouping the delegate methods at the end.

Comment (ryanlntn) [inline lib/ProMotion/iap.rb:127]: Yeah I can see the logic in it. I'll revise.

**Comment by jamonholmgren** [comment]
Much less impressive...but that's okay. :)

**Comment by squidpunch** [comment]
hah - but travis passed!

**Pull request merged**